rabbitmq-native/src/main/java/org/ballerinalang/messaging/rabbitmq/util/ConnectionUtils.java

```diff
@@ -122,6 +125,11 @@ public static Connection createConnection(BMap<BString, Object> connectionConfig
         }
     }
 
+    private static int getTimeValuesInMillis(BDecimal value) {
+        BigDecimal valueInSeconds = value.decimalValue();
+        return (valueInSeconds.multiply(new BigDecimal(1000))).intValue();
```

Comment:
Shall we define a constant for this, instead of creating a new object each time we call this function?